.ci/package.json.jinja2

```diff
@@ -3,4 +3,3 @@
   "version": "{{version}}",
   "dependencies": {{dependencies | jsonify(indent=4, sort_keys=True)}}
 }
-
```

Comment:
Why this change?

Reply:
this patch is applied by CI using this same bear.
I think it is applied because of the option that I've implemented i.e `allow_trailing_blanklines` which has default value `false`
@Makman2 ^ am i right?

Reply:
I dont think so, coz CI needs to be configured in a way that no new file is being edited or made by it, and coala might have confgured CI correctly ;P. Still i am not sure.

Reply:
Yes it does immediately, due to the default value. Imo this is okay :+1: It's probably worth though to mention this inside the commit body :+1: